Assets/Scripts/Controllers/Events/TradeController.cs

```diff
@@ -148,4 +177,16 @@ private Furniture FindRandomLandingPadWithouTrader()
 
         return null;
     }
+
+    private void ShowTradeDialogBox(Job job)
+    {
+        TraderShipController[] traders = TradeShips.Where(x => x.LandingPad == job.tile.Furniture).ToArray();
+        if (traders.Length != 1)
+        {
+            // PANIC!!!
```

Comment:
From my interpretation if the length != 1 then you have multiple traders on a single pad, in which case you should tell a trader to find another pad and throw an error (or at least a warning).